### PR TITLE
Hn 14 setup api

### PR DESCRIPTION
**Pull request by TvanBronswijk**
Before submitting your pull request, consider the following: Does your code...

- [x] Implement functionality
- [x] Contain comments/documentation
- [x] Build without errors
- [x] Contain and pass unit testing
- [ ] Conform to the Style Guide
- [x] Have no merge conflicts with ```master```

When you've checked all of the boxes above your code is up for review. If approved it can be merged with the ```master``` branch.

If you want feedback on your work before having finished your code, add ```[WIP]``` to the title of your PR.

### CODE REVIEW

File: api/src/main/java/nl/fhict/happynews/api/controller/PostController.java [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+     * @return The Posts in JSON.
+     */
+    @RequestMapping(value = "/post/afterdate/{date}", method = RequestMethod.GET, produces = "application/json")
+    public Collection<Post> getPostAfterDate(@PathVariable("date") String date) {
+        SimpleDateFormat sdf = new SimpleDateFormat("EEE MMM d hh:mm:ss z yyyy");
+        Date properdate = null;
+        try {
+            properdate = sdf.parse(date);
+        } catch (ParseException e) {
+            e.printStackTrace();
```
Comment (SvenDub) [inline api/src/main/java/nl/fhict/happynews/api/controller/PostController.java]: Exceptions should be logged using the slf4j logging framework.

File: .idea/runConfigurations/android.xml [hunk 2]
```diff
@@ -1,33 +0,0 @@
-<component name="ProjectRunConfigurationManager">
-  <configuration default="false" name="android" type="AndroidRunConfigurationType" factoryName="Android App" activateToolWindowBeforeRun="false">
```
Comment (DaanTul) [inline .idea/runConfigurations/android.xml]: these deletions shouldn't be pushed i think. Same goes for all runconfigurations

File: .idea/runConfigurations/android.xml [hunk 3]
```diff
@@ -1,33 +0,0 @@
-<component name="ProjectRunConfigurationManager">
```
Comment (SvenDub) [inline .idea/runConfigurations/android.xml]: We don't want to lose this :(

File: .idea/runConfigurations/api.xml [hunk 4]
```diff
@@ -1,18 +0,0 @@
-<component name="ProjectRunConfigurationManager">
```
Comment (SvenDub) [inline .idea/runConfigurations/api.xml]: We don't want to lose this :(

File: .idea/runConfigurations/crawler.xml [hunk 5]
```diff
@@ -1,13 +0,0 @@
-<component name="ProjectRunConfigurationManager">
```
Comment (SvenDub) [inline .idea/runConfigurations/crawler.xml]: We don't want to lose this :(

File: .idea/runConfigurations/web.xml [hunk 6]
```diff
@@ -1,13 +0,0 @@
-<component name="ProjectRunConfigurationManager">
```
Comment (SvenDub) [inline .idea/runConfigurations/web.xml]: We don't want to lose this :(

**Pull request merged**